packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -219,6 +214,7 @@ export abstract class FluidDataStoreContext
 	}
 
 	public get IFluidDataStoreRegistry(): IFluidDataStoreRegistry | undefined {
+		assert(this.channel !== undefined, "registry is populated only when runtime is created");
```

Comment:
No, it's other way around - this function should be called only after channel is created, as otherwise we have not yet populated the registry. This helped me find a bug in my implementation where my proxy parent was grabbing a value (instead of having a getter and thus delaying calling into here until my getter is called).

Reply:
I was talking about the phrasing not the assertion/condition. As a developer, reading "Registry is populated only when runtime is created" tells me there's something wrong with the way or the time when the registry is populated, not about the orchestration of calling this function, which is extra confusing because I'm calling 'get' on the registry, so why is it complaining about when it's modified? I can change the text based on your clarification..